src/SmartComponents/BaselinesPage/EditBaseline/FactModal/FactModal.js

```diff
@@ -225,14 +232,14 @@ export class FactModal extends Component {
                     <Button
                         key="confirm"
                         variant="primary"
-                        ouiaId="save"
+                        ouiaId="save-button"
```

Comment:
I should have asked the same question here. This goes for every button like this. Though, `save-button` may be unique. Do we want to call it `fact-modal-save-button`? It's your call really.